### DISCUSSION TITLE
can juicefs prevent print `#` when cat accesslog?

### DISCUSSION THREAD

**Discussion by qingyunha** [juicedata/juicefs#2184] [General]
print `#` seems useless.

**Comment by davies** [comment]
We print a `#`  every second to tell that the accesslog is live and there is no other operations.

**Comment by qingyunha** [comment]
I thought, for testing accesslog alive, we can simply `ls` the mount point.  The many `#` is some a nosy.

**Reply by davies** [reply]
can you filter the `#` out using `grep -v #`?

**Reply by qingyunha** [reply]
ok 😅
